src/util/account.js

```diff
@@ -42,32 +42,11 @@ export function validateSeed(seed) {
     };
   }
   const words = seed.split(' ');
-  const set = new Set();
-  for (let word of words) {
-    if (typeof WORDS_INDEX[word] === 'undefined') {
-      if (set.has(word)) {
-        return {
-          valid: false,
-          reason: `A duplicated word "${word}" found. Words in seed phrase must be unique.`
-        };
-      }
-      if (word === '') {
-        return {
-          valid: false,
-          reason: `Extra whitespace found`
-        };
-      }
-      return {
-        valid: false,
-        reason: `The word "${word}" is not from the word list`
-      }
-    }
-    set.add(word);
-  }
-  if (set.size < 11) {
+
+  if (words.length < 11) {
```

Comment:
What happened with the previous validation? It seemed quite nice

Reply:
It broke for BIP39 since it uses a different dictionary. This is pretty MVP, we want to add the double-dictionary hinting mechanism that figures out which phrase type you are entering after 1-n words, right now the checks are done in the "backend". I did reintroduce checking for extra whitespace just now though, as that is useful for both.